samples/sample-junit5/junit5.sample.out

```diff
@@ -7,17 +7,18 @@
 
 > Task :test
 
-SampleTests > failing() FAILED
-    org.opentest4j.AssertionFailedError at SampleTests.java:21
 
-SampleTests > flaky() FAILED
+org.gradle.testretry.sample.SampleTests > failing() FAILED
+    org.gradle.internal.serialize.PlaceholderAssertionError at SampleTests.java:21
+
+org.gradle.testretry.sample.SampleTests > flaky() FAILED
     java.lang.RuntimeException at SampleTests.java:16
 
-SampleTests > failing() FAILED
-    org.opentest4j.AssertionFailedError at SampleTests.java:21
+org.gradle.testretry.sample.SampleTests > failing() FAILED
+    org.gradle.internal.serialize.PlaceholderAssertionError at SampleTests.java:21
 
-SampleTests > failing() FAILED
-    org.opentest4j.AssertionFailedError at SampleTests.java:21
+org.gradle.testretry.sample.SampleTests > failing() FAILED
+    org.gradle.internal.serialize.PlaceholderAssertionError at SampleTests.java:21
```

Comment:
@marcphilipp is the internal class `org.gradle.internal.serialize.PlaceholderAssertionError` showing up here ok?

Reply:
Hmm, that's weird indeed. I think it was introduced in 6.6. Maybe there's some special logic in Gradle for the old `PlaceholderException` that's missing for `PlaceholderAssertionError`?

Reply:
Looks like it's caused by this:
https://github.com/gradle/gradle/blob/4cff28254160fe259563d0cffb92229746c092fc/subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/logging/ShortExceptionFormatter.java#L50-L51

Reply:
I've submitted https://github.com/gradle/gradle/pull/13995 to fix this in Gradle.

Reply:
will this fix result in printing `PlaceholderAssertionError` instead of `org.gradle.internal.serialize.PlaceholderAssertionError`?

Reply:
No, it will result in printing `org.opentest4j.AssertionFailedError`.